src/backend/joanie/core/models/products.py

```diff
@@ -313,6 +318,11 @@ class Order(BaseModel):
     All course runs to enroll selected are defined here.
     """
 
+    organization = models.ForeignKey(
+        to=courses_models.Organization,
+        verbose_name=_("organization"),
+        on_delete=models.PROTECT,
+    )
```

Comment:
shouldn't we have a constraint making sure organization are one of the organization of the product?